libs/designer/src/lib/ui/connections/runAfterIndicator/styles.less

```diff
@@ -26,6 +26,23 @@
   display: flex;
 }
 
+.msla-run-after-overflow {
+  box-sizing: border-box;
+  font-family: 'Segoe UI Semibold', sans-serif;
+  font-size: 10px;
+  line-height: 1.4;
+  background-color: #d3d3d3;
```

Comment:
Nit: are you takin in consideration dark theme/mode?

Reply:
good point! Added code for that